exporter.go

```diff
@@ -114,8 +114,7 @@ func (e *Exporter) Export(opts ExportOptions) (platform.ExportReport, error) {
 		return platform.ExportReport{}, err
 	}
 
-	// platform API > 0.5
-	if e.PlatformAPI.Compare(api.MustParse("0.5")) > 0 {
+	if e.PlatformAPI.AtLeast("0.5") {
```

Comment:
According to the previous condition, it should be:
```
e.PlatformAPI.AtLeast("0.6")
```

Reply:
Good catch! Updated in afc8cc9